docs/reference_files.md

```diff
@@ -0,0 +1,47 @@
+# Reference files
+
+## Panel of normals
+
+A pipeline for generating panels of normals for various software is included in Poppy. This leverages the main pipeline when it comes to generating input files for the reference-specific rules, which in turn means that the configuration for the main pipeline in addition to the reference pipeline has to be applied when running it.
+
+As an example, consider the following Snakemake profile (`reference_profile/config.yaml`):
```

Comment:
Feel stupid but this isn't a real file right? Cause i cant find it :sweat_smile:

Reply:
Indeed, it is not a real file. It was just meant as an example, and I needed the file name a bit further down. I'll make that a bit more clear.